### ISSUE TITLE
When deployment fails due to a lock, I'd like to know what task has the lock

### ISSUE THREAD

**Issue by anEXPer** [cloudfoundry/bosh-cli#237]
I recently had a failure in CI where an errand failed to run because of a deployment lock.

```
Director task 305825
Error 100: Failed to acquire lock for lock:deployment:cf-a1 uid: 534d50e4-600d-4126-8706-8e7518be5f1c

Task 305825 error

Errand `smoke_tests' did not complete
```

It would have been nice to get some output explaining what had the lock at the time; it would save me having to dig up env creds and poke around the director, and make forensics easier if by the time I get to the broken build, the lock has already been released.

Could the CLI provide more information about the lock when this happens? Ideally the task number and description?

**Comment by dpb587-pivotal** [comment]
Would `bosh tasks` to list the other running tasks have answered the question without jumping onto the vm?

**Comment by anEXPer** [comment]
That's what I did. I got env creds, did bosh locks, then bosh tasks
--no-filter.

I'd like to have just seen there was a scan and fix happening in the build
output though. (This was happening in Concourse.)

**Comment by cppforlife** [comment]
@anEXPer story: https://www.pivotaltracker.com/story/show/146034015

**Comment by anEXPer** [comment]
That's useful for improving the `bosh locks` command, which I appreciate.

It would be nice if the failed build output told you the task that had the lock, which is what this issue is asking for}.

**Comment by freddesbiens** [comment]
Hi @anEXPer. 

Thanks for this issue. I discussed this with the team, and we feel the cost/benefit of doing this is fairly low. Closing the issue since we will probably not get to it anytime soon.

**Comment by dpb587-pivotal** [comment]
FYI, this did end up being implemented in [v266.2.0](https://github.com/cloudfoundry/bosh/releases/tag/v266.2.0). The newer error messages should provide a message like `Failed to acquire lock for deployment:1234 uid: .... Locking task id is 1234`.
